gammapkt.h

```diff
@@ -7,7 +7,7 @@
 namespace gammapkt {
 void init_gamma_data();
 void pellet_gamma_decay(Packet &pkt);
-void do_gamma(Packet &pkt, double t2);
+void treat_gamma_packet(Packet &pkt, double t2);
```

Comment:
What is the reason for changing this? This should be consistent with do_rpkt, do_ntlepton, do_kpkt, etc.

Reply:
I've just renamed the functions to make it consistent again